crates/store/README.md

```diff
@@ -149,10 +149,11 @@ contains excessive notes and nullifiers, client can make additional filtering of
 
 - `chain_tip`: `uint32` – number of the latest block in the chain.
 - `block_header`: `BlockHeader` – block header of the block with the first note matching the specified criteria.
-- `mmr_delta`: `MmrDelta` – data needed to update the partial MMR from `block_num + 1` to `block_header.block_num`.
-- `accounts`: `[AccountSummary]` – account summaries for accounts updated after `block_num + 1` but not after `block_header.block_num`.
-- `notes`: `[NoteSyncRecord]` – a list of all notes together with the Merkle paths from `block_header.note_root`.
-- `nullifiers`: `[NullifierUpdate]` – a list of nullifiers created between `block_num + 1` and `block_header.block_num`.
+- `mmr_delta`: `MmrDelta` – data needed to update the partial MMR from `request.block_num + 1` to `self.block_header.block_num`.
+- `accounts`: `[AccountSummary]` – account summaries for accounts updated after `request.block_num + 1` but not after `self.block_header.block_num`.
+- `transactions`: `[TransactionSummary]` – transaction summaries for transactions included after `request.block_num + 1` but not after `self.block_header.block_num`.
+- `notes`: `[NoteSyncRecord]` – a list of all notes together with the Merkle paths from `self.block_header.note_root`.
+- `nullifiers`: `[NullifierUpdate]` – a list of nullifiers created between `request.block_num + 1` and `self.block_header.block_num`.
```

Comment:
I would make it clear that each entry here is a tuple `(nullifier, block_num)`.

Reply:
done!